nimble/host/src/ble_l2cap_coc.c

```diff
@@ -29,6 +29,21 @@
 
 #define BLE_L2CAP_SDU_SIZE              2
 
+#define LOG_1(n) (((n) >= 2) ? 1 : 0)
+#define LOG_2(n) (((n) >= 1<<2) ? (2 + LOG_1((n)>>2)) : LOG_1(n))
+#define LOG(n)   (((n) >= 1<<4) ? (4 + LOG_2((n)>>4)) : LOG_2(n))
+/* Log2(TOTAL_CID_PER_MASK_ELEMENT) */
+#define LOG2_TOTAL_CID_PER_MASK_ELEMENT   (LOG(TOTAL_CID_PER_MASK_ELEMENT))
+
+#define SET_CID_MASK(maskBuf,indx)    ((maskBuf)[((indx) >> \
+                                      LOG2_TOTAL_CID_PER_MASK_ELEMENT)] |= \
+                                      (1UL << ((indx) & ((1 << \
```

Comment:
nitpick: be consistent once it is used `1UL << ` and in the same line `1 << `

Reply:
updated